docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+# this is an example of working docker compose file for fcms
+# create an .env file based on the example (don't forget to add the APP_PORT variable)
+version: "3.7"
+services:
+  app:
+    image: ghcr.io/leolivier/fcms:artisan
```

Comment:
ghcr.io is not Google's Registry, it is GitHub's (GHCR = GitHub Container Registry)
That said, the image name is obviously wrong because it's the one I generated myself, it should become `ghcr.io/ryanhowdy/fcms:artisan` as soon as you generate an "official" one. 
Also, if users want to generate their own one, they'll need to update the image name accordingly like I did it myself ;)

Reply:
Well, I disagree with the point of view that users should build the image themselves. If you want to make the life of your users easier, the owner should build the image himself each time there is a new release, put it in a registry (that's easy to do with Github Actions) and users won't have to even clone your github repo (which contains a non released version usually), you just need to provide an example of a docker-compose file like I did and an example of .env file that they can just copy/paste.
And, again, my docker-copose file was just an example and, I fully agree that the image name should be replaced by ryan's one. I will deliver again with his own repo instead of mine... (but it assumes Ryan will deliver an image ;)

Reply:
just did it